dev/buildtool/git_support.py

```diff
@@ -1160,8 +1160,13 @@ def clone_repository_to_path(
         logging.debug("Finished cloning %s", pull_url)
 
     def checkout(self, repository, commit):
+        """Checkout local repository at commit"""
         self.check_run(repository.git_dir, "checkout -q " + commit, echo=True)
 
+    def checkout_branch(self, git_dir, branch, start_point):
```

Comment:
nit: clearer to name this `create_branch` ?

Reply:
Sure, https://github.com/spinnaker/buildtool/pull/193/commits/b6e7925bb46a36c36197bf4124c6b1737e8269e5